lib/EvalSourceMapDevToolModuleTemplatePlugin.js

```diff
@@ -52,12 +51,11 @@ class EvalSourceMapDevToolModuleTemplatePlugin {
 				}, moduleTemplate.requestShortener);
 			});
 			moduleFilenames = ModuleFilenameHelpers.replaceDuplicates(moduleFilenames, (filename, i, n) => {
-				for(let j = 0; j < n; j++)
-					filename += "*";
+				for (let j = 0; j < n; j++) filename += "*";
```

Comment:
```js
for (let j = 0; j < n; filename += "*", j++);
```
:trollface:

Reply:
Couldn't you just `return filename + "*".repeat(n);` ?

Reply:
repeat didn't exist when this code was written... ;)